mode_queries/parameters.sql

```diff
@@ -0,0 +1,18 @@
+select 1
+
+{% form %}
```

Comment:
can you put this form section into `audience_overview.sql`? we might want to add other queries in the future that could potentially have different parameters.